### ISSUE TITLE
Entire system crash from the interpreter

### ISSUE THREAD

**Issue by LoganDark** [NodeOS/NodeOS#420]
`require('fs').readFileSync('/bin/node').toString('utf8')`, in the interpreter, crashes the entire system.

There's a bunch of extremely fast scrolling text and then the system immediately powers off.

**Comment by piranna** [comment]
Obviously, bin/node is the Node.js executable, not only is binary but also is really big, so it's normal it output garbage. It should not crash, this could be for an out of memory problem, or some instruction inside the Node.js binary itself. Can you provide more info about how it crash?

**Comment by LoganDark** [comment]
@piranna The screen floods with text very quickly and then the system immediately shuts off, check the issue.

> it's normal it output garbage

The text is by the kernel (i.e. has timestamps and ASCII and all that).
